bin/run_server.py

```diff
@@ -0,0 +1,154 @@
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import optparse
+import sys
+
+from kmip.services import server
+
+
+def build_argument_parser():
+    parser = optparse.OptionParser(
+        usage="%prog [options]",
+        description="Run KMIP client operation")
+
+    parser.add_option(
+        "-n",
+        "--hostname",
+        action="store",
+        type="str",
+        default=None,
+        dest="hostname",
+        help=(
+            "A string representing either a hostname in Internet domain "
+            "notation or an IPv4 address. Defaults to None."
+        ),
+    )
+    parser.add_option(
+        "-p",
+        "--port",
+        action="store",
+        type="int",
+        default=None,
+        dest="port",
+        help=(
+            "An integer representing a port number. Recommended to be "
+            "5696 according to the KMIP specification. Defaults to None."
+        ),
+    )
+    parser.add_option(
+        "-c",
+        "--certificate_path",
+        action="store",
+        type="str",
+        default=None,
+        dest="certificate_path",
+        help=(
+            "A string representing a path to a PEM-encoded server "
+            "certificate file. Defaults to None."
+        ),
+    )
+    parser.add_option(
+        "-k",
+        "--key_path",
+        action="store",
+        type="str",
+        default=None,
+        dest="key_path",
+        help=(
+            "A string representing a path to a PEM-encoded server "
+            "certificate key file. Defaults to None."
+        ),
+    )
+    parser.add_option(
+        "-a",
+        "--ca_path",
+        action="store",
+        type="str",
+        default=None,
+        dest="ca_path",
+        help=(
+            "A string representing a path to a PEM-encoded certificate "
+            "authority certificate file. Defaults to None."
+        ),
+    )
+    parser.add_option(
+        "-s",
+        "--auth_suite",
+        action="store",
+        type="str",
+        default=None,
+        dest="auth_suite",
+        help=(
+            "A string representing the type of authentication suite to use "
+            "when establishing TLS connections. Defaults to None."
+        ),
+    )
+    parser.add_option(
+        "-f",
+        "--config_path",
+        action="store",
+        type="str",
+        default=None,
+        dest="config_path",
+        help=(
+            "A string representing a path to a server configuration file. "
+            "Defaults to None."
+        ),
+    )
+    parser.add_option(
+        "-l",
+        "--log_path",
+        action="store",
+        type="str",
+        default=None,
+        dest="log_path",
+        help=(
+            "A string representing a path to a log file. Defaults to None."
```

Comment:
`stdout` is not a supported option right now, as it is handled differently by the Python `logging` utility. The `KmipServer` only supports file-based logging at the moment.